tests/test_infer/test_bloom_infer.py

```diff
@@ -0,0 +1,71 @@
+import os
+
+import pytest
+import torch
+import torch.distributed as dist
+from packaging import version
+from transformers import AutoTokenizer, BloomForCausalLM
+
+import colossalai
+from colossalai.inference.tensor_parallel import TPInferEngine
+from colossalai.logging import disable_existing_loggers
+from colossalai.testing import clear_cache_before_run, parameterize, rerun_if_address_is_in_use, spawn
+
+TP_SIZE = 2
+MAX_BATCH_SIZE = 4
+MAX_INPUT_LEN = 16
+MAX_OUTPUT_LEN = 32
+
+CUDA_SUPPORT = version.parse(torch.version.cuda) > version.parse('11.5')
+
+
+@parameterize('test_config', [{
+    'tp_size': TP_SIZE,
+}])
+def run(test_config):
+    model_path = "/data3/models/bloom-7b1"
```

Comment:
Don't use hardcoded path. If necessary, use env variable and define it in workflow.

Reply:
By the way, is it this model too larg? How long does this test take?

Reply:
(Bloom-7B) For local testing, it takes about less than a minute

Reply:
Test revised and path removed.